packages/matrix/docker/synapse/templates/registration.txt

```diff
@@ -0,0 +1,5 @@
+Please confirm that {{ email }} is your e-mail address by clicking this link within 48 hours.
```

Comment:
1 hour

Reply:
Or just leave out the time component. If the link becomes invalid the user will be informed about that right?

Reply:
There's no time config we could read in the email registration template. I think I'll leave it out.